frontend/destino-cordoba/src/app/components/registro/registro.component.html

```diff
@@ -0,0 +1,66 @@
+<div class="container">
+  <h1>Bienvenido!</h1>
+  <br />
+  <div class="registro">
+    <h2>Registrate</h2>
+
+    <form id="registro" method="post" action="/endpoint">
+      <label for="nombre">Nombre:</label>
+      <input
+        type="text"
+        id="nombre"
+        name="nombre"
+        required
+        [formControl]="nombre"
+      /><br /><br />
```

Comment:
```suggestion
      />
```
Mejor si no usas los br, es mejor un margin-bottom o un gap entre los elementos